src/screens/Home.js

```diff
@@ -109,41 +129,41 @@ export default ({
             />
           </View>
           <View style={styles.heading}>
-            <SectionHeading text="Select a category:" />
+            <SectionHeading text={selectCatText} />
           </View>
           <List
             lang={nativeLanguage}
             data={categories}
-            text={'Learn'}
+            text={learnText}
             color="#06B6D4"
             iconType="material-community"
             iconName="arrow-right"
             makeAction={openCategoryPhrases}
           />
           <View style={styles.heading}>
-            <SectionHeading text="Seen phrases:" />
+            <SectionHeading text={seenPhraseText} />
           </View>
           <List
-            data={[{id: 1, name: '35 words and phrases'}]}
-            text={'Learn'}
+            data={[{id: 1, name: `35 ${wordsAndPhrases}`}]}
```

Comment:
Is this really necessary?

Reply:
I have removed the repeated lines I just keep this one data={[{id: 1, name: `35 ${wordsAndPhrases}`}]}